controllers/solrcloud_controller.go

```diff
@@ -167,6 +167,21 @@ func (r *SolrCloudReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 				}
 			}
 		}
+	} else {
+		// Need no individual services per onde, delete if found
+		for _, nodeName := range solrNodeNames {
+			serviceName := nodeName
+			foundService := &corev1.Service{}
+			err = r.Get(ctx, types.NamespacedName{Name: serviceName, Namespace: instance.Namespace}, foundService)
+			if err == nil {
+				err = r.Delete(ctx, foundService)
```

Comment:
We would likely want to log here that we are deleting the service.

Reply:
Added logging both for service deletion and ingress deletion